conbench/templates/index.html

```diff
@@ -2,24 +2,7 @@
 
 {% block app_content %}
 
-<div class="p-3 mb-4 bg-light rounded-3">
-  <div class="container-fluid py-2">
-    <div class="row mb-3">
-      <div class="col-md-5">
-        <h1 class="display-5 fw-bold">Con·bench</h1>
-        <p class="col-md-8 fs-4">/kənˈben(t)SH/ <br /><i>noun</i></p>
-      </div>
-      <div class="col-md-7">
-        <div class="float-end">
-          <img class="media-object" style="padding: 5px" src="{{ url_for('static', filename='bar-graph.png') }}" height="180">
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
```

Comment:
I like the link in the footer! But I do miss the jumbotron. 

To your point, efficient use of space is good. But with this change, I feel a little overwhelmed by the amount of text immediately presented on the landing page. There's something to be said about balanced "blankness" being a good use of space too, especially on the main page of the app.

I prefer keeping the jumbotron. But I showed this change to my wife, who's never seen Conbench before, and she agreed with you that "less scrolling is better." So perhaps I'm in the minority here 🙂 Though we still may want to think about what else we could put at the top so that the table looks more balanced (maybe something more useful than just a logo).

Reply:
One way I get utility from the jumbotron was knowing instantly I'm at a home page—a lot of our pages are really just tables, and the jumbotron makes that particular one distinctive. But there are probably more useful ways to do that than with a logo, and we should figure out what some are.

Reply:
>  and she agreed with you that "less scrolling is better."

haha, yes!

From a very puristic, functional point of view the jumbotron is a UX 'failure' :-). I know so much. 

> Though we still may want to think about what else we could put at the top so that the table looks more balanced (maybe something more useful than just a logo).

Absolutely! UX-wise, we achieve perfection when no element can be taken away. Removing the jumbotron is just a quick idea towards breathing, towards making space for better ideas. The jumbotron is non-interactive, so this maybe is thought-provoking -- in _this_ geometrical region of the interface I think we totally want to offer interactivity.

Reply:
> I like the link in the footer! 

OK! :) :foot: 

>  But I do miss the jumbotron.

Now I consider just adding it back for now. Hm.
Or maybe just adding a bit of whitespace?

I realize I have a little bit of this minimization/removal quest going on -- which elements in the UI are here for what? If something isn't really needed, the inclination is to remove it (for now!).

Let's be in flux, in flow -- remove, remove, and then maybe add later again -- without overthinking too much for now.

Higher-level plan: let's be mindful of which navigation/interaction elements we _need_, and which workflows we want to enable. We _can_ at the same time also pay attention to aesthetics and design. But for a UI designer what's a great entry point is to see a well thought-through UX already. We can (and should!) bring in someone (I have someone on the back of my mind :-)) once we are confident that the UX is reasonably well thought-out.